docs/pyroute2-dhcp-client.rst

```diff
@@ -4,49 +4,79 @@ pyroute2-dhcp-client
 Synopsis
```

Comment:
@etene pls add the `Author` section, check the options, and think if we can autogenerate && mock test this doc.